compiler_opt/distributed/buffered_scheduler.py

```diff
@@ -80,3 +81,32 @@ def chain_work(wkr: T):
       chain_work(w)
 
   return results
+
+
+def schedule_on_worker_pool(
+    action: Callable[[W, T],
+                     Any], jobs: Iterable[T], worker_pool: worker.WorkerPool
```

Comment:
maybe pass through a `**kwargs` argument to `schedule` so that the user can set the `buffer` kwarg if they want (or just add a single `buffer` argument, but then if there are any more arguments added to `schedule` in the future we'd also have to hook them through here)

Reply:
done, exposing it explicitly.